decent-dao/fractal-interface
# Add remove owners

## Description
This PR:
1) Adds a "Manage Signers" option on the "Manage DAO" menu option
2) Adds a page listing the current safe signers, and buttons for adding and removing signers
3) Adds modals for both adding and removing signers, which allows a user to create a proposal to do either action

![image](https://user-images.githubusercontent.com/84364476/231892261-738b8b69-8df7-4d4a-88a4-d3da3aff9b94.png)
![image](https://user-images.githubusercontent.com/84364476/231892305-8163ee50-c980-4bd7-84a7-cb152be751d2.png)
![image](https://user-images.githubusercontent.com/84364476/231892443-e09d7760-208a-4d90-a452-8db2490f8e16.png)
![image](https://user-images.githubusercontent.com/84364476/231892486-eebba4ce-5c5d-4f76-9bd8-f48086f556d0.png)



## Notes



## Issue / Notion doc (if applicable)



## Testing
1) Create a proposal to add a signer, execute the proposal and verify that the signer was added to the safe
2) Create a proposal to remove a signer, execute the proposal and verify that the signer was removed from the safe


## Screenshots (if applicable)
